packages/@lightningjs/ui-components/src/components/TextMagnifier/TextMagnifier.stories.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import lng from '@lightningjs/core';
+import { default as TextMagnifierComponent } from '.';
+import { context } from '@lightningjs/ui-components/src';
+
+export default {
+  title: 'Components/TextMagnifier'
+};
+
+export const TextMagnifier = () =>
```

Comment:
Can you change this const to "Basic" to match our other stories? Doing this seems to not nest it in a folder like the other components.